src/main/java/org/ow2/proactive/catalog/rest/controller/BucketController.java

```diff
@@ -206,7 +207,7 @@ public List<BucketMetadata> list(
             @ApiParam(value = "The name of objects that buckets must contain") @RequestParam(value = "objectName", required = false) Optional<String> objectName,
             @ApiParam(value = "The bucket name contains the value of this parameter (case insensitive)")
             @RequestParam(value = "bucketName", required = false)
-            final Optional<String> bucketName, @ApiParam(value = "If true, buckets without objects matching the filters will be returned with objectCount=0. Default is false") @RequestParam(value = "allBuckets", required = false, defaultValue = "false") String allBuckets) throws NotAuthenticatedException, AccessDeniedException {
+            final Optional<String> bucketName, @ApiParam(value = "The project name equals this parameter (case insensitive)") @RequestParam(value = "projectName", required = false) Optional<String> projectName, @ApiParam(value = "The user who last committed changes in the buckets") @RequestParam(value = "username", required = false) Optional<String> lastCommittedBy, @ApiParam(value = "The time after the last commit took place in the buckets in ms") @RequestParam(value = "committedTimeGreater", required = false, defaultValue = "0") Optional<Long> committedTimeGreater, @ApiParam(value = "The time before the last commit took place in the buckets in ms") @RequestParam(value = "committedTimeLessThan", required = false, defaultValue = "0") Optional<Long> committedTimeLessThan, @ApiParam(value = "If true, buckets without objects matching the filters will be returned with objectCount=0. Default is false") @RequestParam(value = "allBuckets", required = false, defaultValue = "false") String allBuckets) throws NotAuthenticatedException, AccessDeniedException {
```

Comment:
The user who last committed changes in the buckets
=> this is a strange filter honestly

Reply:
The time after the last commit took place in the buckets in ms
=> filters only buckets where the last commit occurred after the given EPOCH time.
The time before the last commit took place in the buckets in ms
=> filters only buckets where the last commit occurred before the given EPOCH time (this filter is a bit strange as it will return any bucket where there exists a least one old commit which will likely be "all buckets")